sdk/core/azure-core/src/main/java/com/azure/core/util/Context.java

```diff
@@ -13,7 +13,9 @@
 /**
  * {@code Context} offers a means of passing arbitrary data (key-value pairs) to pipeline policies.
  * Most applications do not need to pass arbitrary data to the pipeline and can pass {@code Context.NONE} or
- * {@code null}. Each context object is immutable. The {@code addData(Object, Object)} method creates a new
+ * {@code null}.
+ * <p>
+ * Each context object is immutable. The {@code addData(Object, Object)} method creates a new
```

Comment:
You can use `{@link #addData(Object, Object)}`. If the method gets renamed, this reference also gets renamed.